src/Build.OM.UnitTests/Microsoft.Build.Engine.OM.UnitTests.csproj

```diff
@@ -55,6 +55,9 @@
     <Compile Include="..\Shared\ResourceUtilities.cs">
       <ExcludeFromStyleCop>true</ExcludeFromStyleCop>
     </Compile>
+    <Compile Include="..\Shared\Traits.cs">
+      <Link>Traits</Link>
```

Comment:
Does this make it look like it's SharedTraits.cs inside VS? That might be confusing :)

---
In reply to: [116095729](https://github.com/Microsoft/msbuild/pull/2087#discussion_r116095729) [](ancestors = 116095729)